tests/integration/main.rs

```diff
@@ -0,0 +1,111 @@
+mod artichoke;
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
```

Comment:
```suggestion
use std::process::{Command, Output};
```

Can we import this type which gets used below to avoid using a fully qualified item path?